bundles/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/handler/GardenaAccountHandler.java

```diff
@@ -157,11 +279,7 @@ public Collection<Class<? extends ThingHandlerService>> getServices() {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        if (RefreshType.REFRESH == command) {
-            logger.debug("Refreshing Gardena account '{}'", getThing().getUID().getId());
-            disposeGardena();
-            initializeGardena();
-        }
+        // nothing to do here because the thing has no channels
     }
```

Comment:
Can simply drop that method override then.

Reply:
Nope. It is an abstract method in the superior class, so it must be overridden.

Reply:
OK, TIL :+1: